renku/core/commands/providers/renku.py

```diff
@@ -57,7 +57,13 @@ def find_record(self, uri, client=None):
         failed_urls = []
 
         for kg_url in kg_urls:
-            kg_datasets_url, ssh_url, https_url = self._get_project_urls(kg_url)
+            try:
+                kg_datasets_url, ssh_url, https_url = self._get_project_urls(kg_url)
+            except errors.OperationError as e:
+                # NOTE: Project was likely deleted, but still referenced in the KG
+                if "project not found" not in e.message:
+                    raise
+                failed_urls.append(kg_url)
```

Comment:
It's better to have a separate error list (e.g. `non_existing_projects`) and then have a separate error message for them.

Reply:
In this case, it should be possible to have a failing test that tries to import a non-existing project and then check for the proper error message.

Reply:
We should also ` continue` at this point to avoid running the rest of the code outside the try/except block.